init.lua

```diff
@@ -114,7 +114,12 @@ function module.focus_client(c,properties)
         local tags = c:tags()
 
         if #tags > 0 and not has_selected(tags, c.screen) and not tags[1].no_focus_stealing_in then
-            c:tags()[1]:view_only()
+            t = c:tags()[1]
+            if t.no_focus_only then
+                t.selected = true
```

Comment:
I think this should use `c:view_more()`. If, for example, someone implements a Tyrannical style module to manage moving across tags, using `t.selected = true` would bypass whatever logic is hooked there.

Reply:
`c:view_more()` here gives me a "attempt to call a nil value (method view_more())", what am I doing wrong?

Reply:
oops, sorry, it's called `awful.tag.viewmore(c:tags())`, I should fix that, it's confusing.

Reply:
Well then I would have to supply the currently selected tags as well, according to the [documentation of that function](https://awesomewm.org/doc/api/classes/tag.html#awful.tag.viewmore), which could be done quite easily, but then the question arises if that would not also be hardcoded behavior and therefore the simpler `t.selected = true` be preferable. I don't have a strong opinion there, your call. :)

Reply:
Ok, I will fix it once it becomes a real problem

Reply:
So we go with `t.selected` and I add a comment that this is hardcoded behavior?

Reply:
sure, it's not a problem until such behavior is added. Sorry for the confusion I caused on this. Fixing the other 2 issues will be plenty.